### ISSUE TITLE
Multiple PDF Files

### ISSUE THREAD

**Issue by kalyanramu** [bhaskatripathi/pdfGPT#36]
What would it take to support Multiple PDF File Import and providing search capability?

**Comment by bhaskatripathi** [comment]
Yes ofcourse that can be done within pdfGPT. Have added this item to the backlog for version 2.

**Comment by Sassun** [comment]
+1 for multiple PDF files. Would be also amazing if it would be possible to add other file formats as well .docx, HTML etc.

**Comment by kalyanramu** [comment]
> Yes ofcourse that can be done within pdfGPT. Have added this item to the backlog for version 2.

Nice, I am curious to hear the plan to add [PDF name along with line number]

**Comment by flake9** [comment]
+1

**Comment by ollieanwyll** [comment]
+1

**Comment by bhaskatripathi** [comment]
have created a very crude version of multiple-pdf support but it may not be the best code as I know some known issues.
In the mean while you can check the sandbox environment here:
https://huggingface.co/spaces/bhaskartripathi/pdfChatterSandbox

**Comment by liugddx** [comment]
+1. I need it badly.

**Comment by greenTea2Codes** [comment]
Hi, great tool! Is it possible to upload a collection of articles on a particular subject ONCE and instruct the tool to always return answers from the given set of articles with references?

**Comment by faburem** [comment]
I think multiple PDF support could be an outstanding feature. I can imagine this for software release notes published only as PDF by vendors or "old school" eLearning material. Imagine you could turn all that into an interactive knowledge base! This could really be a huge deal for some companies.

**Comment by bhaskatripathi** [comment]
I am working on it. Will soon release this feature.
